tests/unit/fixtures/QuoteMessage.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
If each plugin had its own folder in `fixtures/`, it would be more neatly organized, especially when the number of fixtures (and especially fixtures per plugin) gets large. Consider moving it to `fixtures/Quote/Message.json', `fixtures/plugins/Quote/Message.json` or something like that.